script.js

```diff
@@ -1,43 +1,88 @@
+/* eslint-disable max-classes-per-file */
 class Books {
-    constructor(title, author) {
-        this.title = title;
-        this.author = author;
-        this.BookId = this.BookId;
-    }
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
 
-const bookArchive = [];
-const form  = document.querySelector('.form');
+const form = document.querySelector('.form');
 const booksDiv = document.querySelector('.books');
 
+class CreateNewBook {
+  static addNewBook(book) {
+    const bookUnit = document.createElement('li');
+    bookUnit.innerHTML = `
+        <p class="book-name">${book.title}</p>
+        <p class="the-auhtor">${book.author}</p>
+        <button class='removeBook' id=${Books.id}>Remove</button>
+        `;
+    booksDiv.appendChild(bookUnit);
+    booksDiv.style.border = '3px solid black';
+  }
 
+  static removeBookFromPage(target) {
+    if (target.classList.contains('removeBook')) {
+      target.parentElement.remove();
+    }
 
+    if (!booksDiv.firstElementChild) {
+      booksDiv.style.border = '3px solid white';
+    }
+  }
 
-form.addEventListener('submit', (event) => {
-    event.preventDefault();
+  static loadFromStorage() {
+    let books;
 
-    const title = document.querySelector('.book-title').value;
-    const author = document.querySelector('.book-author').value;
+    if (localStorage.getItem('bookInfo')) {
+      books = JSON.parse(localStorage.getItem('bookInfo'));
+    } else {
+      books = [];
+    }
 
-    const book = new Books (title, author)
-    bookArchive.push(book);
-    addNewBook(book)
+    return books;
+  }
 
-})
+  static displayBooksFromStorage() {
+    const books = CreateNewBook.loadFromStorage();
 
+    books.forEach((book) => {
+      CreateNewBook.addNewBook(book);
+    });
+  }
 
-function addNewBook(book) {
-    booksDiv.innerHTML = `
-    <p class="book-name">${book.title}</p>
-    <p class="the-auhtor">${book.author}</p>
-    <button class='removeBook' >Remove</button>
-    `
+  static removeBookFromStorage(element) {
+    const books = CreateNewBook.loadFromStorage();
+    const title = element.parentElement.firstElementChild.innerHTML;
+    const index = books.findIndex((book) => book.title === title);
+    books.splice(index, 1);
+
+    localStorage.setItem('bookInfo', JSON.stringify(books));
+  }
```

Comment:
- [ ] @coderacheal and @EnixWord great work on your function, however when I have books with same title, the delete fucntion always deletes the wrong book, you can add  unique id for the books to prevent this bug.